content/docs/dvclive/api-reference/plots.md

```diff
@@ -0,0 +1,214 @@
+# Plotting utilities
+
+Generates
+[Scikit-learn visualizations](https://scikit-learn.org/stable/visualizations.html)
```

Comment:
```suggestion
[scikit-learn visualizations](https://scikit-learn.org/stable/visualizations.html)
```

Reply:
I committed most of these but this one (and another similar one) I'd also capitalize since Scikit-learn is a proper noun. Or we could go with the brand's preferred lower-case `scikit-learn` everywhere (like we do with "GitHub" in DVC docs at least). Ultimately as long as it's consistent either way works so up to you @daavoo